packages/react-popover/src/components/PopoverContent/usePopoverContent.ts

```diff
@@ -31,6 +31,8 @@ export const usePopoverContent = (
   const noArrow = usePopoverContext(context => context.noArrow);
   const brand = usePopoverContext(context => context.brand);
   const inverted = usePopoverContext(context => context.inverted);
+  const trapFocus = usePopoverContext(context => context.trapFocus);
+  const { modalAttributes } = useModalAttributes({ trapFocus });
```

Comment:
not currently, this maps onto tabster `deloser` and `modalizer` . I don't see any sense currently to allow the use of `deloser` because it can be abused. 

Keeping focus history is not free, so let's not make it easy